Observer/Product/AbstractChangedProductObserver.php

```diff
@@ -99,7 +100,7 @@ public function execute(Observer $observer)
                 }
                 
                 if ($product->getStore()->getId() == 0 ||
-                    $this->getOperationType() == ChangedItemInterface::OPERATION_TYPE_DELETE
+                    $operationType == ChangedItemInterface::OPERATION_TYPE_DELETE
```

Comment:
This change is intended?
Seems that the operation type updated in the if above won´t be taken into account on this if, and seems like it should

Reply:
no entiendo bien esto, lo he cambiado porque en todos los otros casos se hace la comparación directamente con `operationType` porque se define arriba `$operationType = $this->getOperationType();` y era por tener todos igual

Reply:
gracias!! no lo había visto